packages/react/src/patterns/sub-patterns/Quote/__stories__/Quote.stories.js

```diff
@@ -0,0 +1,63 @@
+import './index.scss';
+
+import {
+  text,
+  withKnobs,
+  select,
+  object,
+  boolean,
+} from '@storybook/addon-knobs';
+import Quote from '../Quote';
+import React from 'react';
+import readme from '../README.md';
+import { storiesOf } from '@storybook/react';
+
+storiesOf('Patterns (Sub-Patterns)|Quote', module)
+  .addDecorator(withKnobs)
+  .addParameters({
+    readme: {
+      sidebar: readme,
+    },
+  })
+  .add('Default', () => {
+    const copy = text(
+      'Quote(Required): ',
```

Comment:
a format I started following is to include the prop name in parenthesis, might be worth following (Carbon seems to do this too now):

```
const copy = text('quote text (copy)', 'Lorem Ipsum');
const markType = select('quotation mark type (markType):', types, types.doubleCurved);
...etc...
```

Reply:
I've updated it on the latest commit